Misc/NEWS.d/next/Library/2020-02-18-12-37-16.bpo-39479.j3UcCq.rst

```diff
@@ -0,0 +1 @@
+Add :func:`math.lcm` function: Least Common Multiple.
```

Comment:
It doesn't matter that much, since this is only a news entry, but why the capitalization? This wasn't there in your original, approved PR.